package.json

```diff
@@ -21,23 +21,24 @@
   "types": "node/lib/index.d.ts",
   "files": [
     "node/lib",
-    "worker/deps/libwebrtc",
+    "npm-scripts.mjs",
```

Comment:
More canonical order it to have directories first and files later, this is how it is typically sorted (on GitHub, in file manager, etc.). Maybe keep it that way?

Reply:
Actually I thought it was not ordered that way. I'll honor it.